test/integration/apiserver/crd_validation_expressions_test.go

```diff
@@ -300,6 +300,78 @@ func TestCustomResourceValidators(t *testing.T) {
 			t.Error("Unexpected error creating custom resource but metadata validation rule")
 		}
 	})
+	t.Run("Schema with valid transition rule", func(t *testing.T) {
+		structuralWithValidators := crdWithSchema(t, "Structural", structuralSchemaWithValidTransitionRule)
```

Comment:
oh, not that weird... need to pick a unique CRD name here

Reply:
Ah, that's my integration test failure. Thx!